snippets/csharp/System.Net.Http/WinHttpHandler/Project.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Can it target .NET 9 instead?

Reply:
Of course it can. I just copy pasted from another example, so I thought there's some reason for it being stuck at 6.0.